docs/data/charts/quickstart/quickstart.md

```diff
@@ -130,3 +138,10 @@ const theme = createTheme({
   },
 });
 ```
+
+## Using this documentation
+
+Most Chart components have two accompanying documents:
+
+1. **Overview** – a general description of built-in features
+2. **Demos** – a collection of custom examples
```

Comment:
Not sure how necessary this section is, as it can be deduced from reading the sidebar

Reply:
True, probably don't really need to spell this out. "Using this documentation" is a helpful section for the other products but maybe less useful here. Unless there's anything else you can think of that the reader should know about the docs before continuing past this page?

Reply:
I can't think of any extra info the user should `need` to know.